Rakefile

```diff
@@ -106,11 +106,6 @@ SCENARIOS.each do |scenario|
       'parser_cache_path' => "#{DATADIR}/foreman-installer/parser_cache/#{scenario}.yaml",
     }
 
-    # These scenarios have their own hook dirs for now
-    if ['foreman-proxy-content', 'katello'].include?(scenario)
-      scenario_config_replacements['installer_dir'] += '/katello'
```

Comment:
This should now be replacing the hook dirs. Since it's an array, sed might be harder to use.

Reply:
Technically, the migrations handle this for us in the end so having it in this build step is not needed. I'm trying to combine the hooks to a single directory anyway which would then remove any hook_dir updating code I added.